pipelines/broad/arrays/validate_chip/ValidateChip.wdl

```diff
@@ -324,7 +324,7 @@ task GenotypeConcordance {
   >>>
 
   runtime {
-    docker: "us.gcr.io/broad-gatk/gatk:4.2.6.1"
+    docker: "us.gcr.io/broad-gatk/gatk:4.6.1.0.6.1"
```

Comment:
It looks like some of these replacements got inserted instead of replaced (this should 4.6.1.0). I'll try to note where I spotted them, but I think there are a few.

Reply:
lol! I don't know what happened 😂 fixing these up now. thanks for pointing them out!